metatron-app/metatron-discovery
# F#1074 create json file type snapshot

### Description
Currently, we only support CSV file type snapshot generating.
JSON file type snapshot generating feature is required.

**Related Issue** : 
https://github.com/metatron-app/metatron-discovery/issues/1074

### How Has This Been Tested?
1. Create snapshot with JSON format to local and hdfs storage.
2. Check if snapshot preview is displayed well.
3. Download generated snapshot file.(Currently it downloaded as csv file. It will be fixed soon.)


### Types of changes



### Checklist:



### Additional Context
